install/kubernetes/cilium/templates/hubble/tls-helm/relay-client-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.hubble.enabled .Values.hubble.tls.enabled .Values.hubble.tls.auto.enabled (eq .Values.hubble.tls.auto.method "helm") .Values.hubble.relay.enabled }}
+{{- $_ := include "hubble-generate-certs.helm.setup-ca" . -}}
+{{- $cn := "*.hubble-relay.cilium.io" }}
+{{- $cert := genSignedCert $cn nil nil (.Values.hubble.tls.auto.certValidityDuration | int) .ca -}}
```

Comment:
Missing SANs:
```suggestion
{{- $cert := genSignedCert $cn nil (list $cn) (.Values.hubble.tls.auto.certValidityDuration | int) .ca -}}
```

Reply:
Actually client certificate is used to authenticate to server. It's no need to specify SANs.